### DISCUSSION TITLE
Best way to perform sql operations?

### DISCUSSION THREAD

**Discussion by ghost** [nuxt/nuxt#20339] [Questions] [answered]
Hey y'all,

I was looking at the database modules on the nuxt websites but I didn't see a module to work with mysql / mariadb,
I was wondering if y'all have sql npm package recommandations ? (or maybe even ORM?) that would run serverside

**Comment by wobsoriano** [comment, marked as answer]
You can use whatever you want.

Also, shameless plug:

1. https://github.com/wobsoriano/numix
2. https://github.com/wobsoriano/nuxt-remote-fn
